### PR TITLE
Players can see each other

### LINKED ISSUE #256: Players can't see other players
Hello we are a multiplayer server but when i connect 2 accounts i'm still alone, is it some kind of reflection of my own life ?

### PR DESCRIPTION
**Pull request by DeltaEvo**
Fix #256

### CODE REVIEW

File: src/player/sync.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+  if (nullIdx === -1) {
+    return { array: [...array, value], index: array.length }
+  } else {
+    return {
+      array: [...array.slice(0, nullIdx), value, ...array.slice(nullIdx + 1)],
+      index: nullIdx,
+    }
+  }
+}
```
Comment (Sceat) [inline src/player/sync.js:21]: why not using a Set instead of manipulating arrays ?

**Review by Sceat — APPROVED** [review]

File: src/player/sync.js (hunk 1 above)
Comment (DeltaEvo) [reply src/player/sync.js:21]: Because I need stable indexes to use as entity id

Comment (Sceat) [reply src/player/sync.js:21]: should'nt we rethink this id management ? random instead of increments

Comment (DeltaEvo) [reply src/player/sync.js:21]: We could, but I don't think it will be simpler :thinking:

**Pull request merged**